core/dbt/parser/schema_test_builders.py

```diff
@@ -334,10 +351,13 @@ def build_raw_sql(self) -> str:
         )
```

Comment:
We need to reorder `"{config}{{{{ {macro}(**{kwargs_name}) }}}}"` to be `{{{{ {macro}(**{kwargs_name}) }}}}{config}`, so that specific configs (supplied by the user in the modifiers) override the generic configs set in the test macro.
```python
  # this is the 'raw_sql' that's used in 'render_update' and execution
  # of the test macro
  # config needs to be rendered last to take precedence over default configs
  # set in the macro (test definition)
  def build_raw_sql(self) -> str:
      return (
          "{{{{ {macro}(**{kwargs_name}) }}}}{config}"
      ).format(
          macro=self.macro_name(),
          config=self.construct_config(),
          kwargs_name=SCHEMA_TEST_KWARGS_NAME,
      )
```